roles/openshift_master/tasks/main.yml

```diff
@@ -25,7 +25,7 @@
       cluster_method: "{{ openshift_master_cluster_method | default(None) }}"
       cluster_hostname: "{{ openshift_master_cluster_hostname | default(None) }}"
       cluster_public_hostname: "{{ openshift_master_cluster_public_hostname | default(None) }}"
-      debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level) }}"
+      debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') | default(openshift.common.debug_level, True) }}"
```

Comment:
This should remain using openshift_master_debug_level.

The lookups in the playbooks/*/openshift-cluster/config.yml playbooks are already setting openshift_master_debug_level to the value from the lookup, and we don't really want the roles to have dependencies on environment variables or cli driven variables.